File: tuan01/tong.jl
function main()
    input_data = split(read(stdin, String))
    if isempty(input_data)
        return
    end
    
    n = parse(Int, input_data[1])
    a = [parse(Int64, input_data[i]) for i in 2:(n+1)]
    
    println("$(sum(a)) $(maximum(a))")
end

main()

File: tuan01/tong.py
import sys

def main():
    input_data = sys.stdin.read().split()
    if not input_data:
        return
    
    n = int(input_data[0])
    a = [int(x) for x in input_data[1:n+1]]
    
    print(f"{sum(a)} {max(a)}")

if __name__ == "__main__":
    main()